Core/GDCore/Extensions/Builtin/AdvancedExtension.cpp

```diff
@@ -80,9 +71,9 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsAdvancedExtension(
                       "a string, an empty string is considered as \"false\". "
                       "If it's a number, 0 is considered as \"false\"."),
                     _("Parameter _PARAM0_ is true"),
-                    _("Functions"),
-                    "res/function24.png",
-                    "res/function16.png")
+                    "",
+                    "res/function32.png",
+                    "res/function32.png")
```

Comment:
why have you changed some instructions icons to 32, and not others?

Reply:
Ideally I would have changed them all to a svg file ;) The reason is that the old IDE was working only with PNG. Now we could have single SVG icon everywhere. At least we're not forced to feed the IDE with two version of the PNG, as the browser will resize them when displaying them (the old IDE needed a specific size). So I just changed as I went some icons to use the larger version we have.